main.py

```
@@ -1,240 +1,195 @@
 import ell
+from ell import Field
 from openai import Client
 import os
 import argparse
+import subprocess
+from pydantic import BaseModel, Field
+from refined_unit_tests import refined_unit_tests
+from pydantic import ValidationError
+import time
+from tools import *
```

Ruff

`from tools import *` used; unable to detect undefined names

(F403)